src/main/java/org/javarosa/core/reference/ReferenceHandler.java

```diff
@@ -39,4 +39,9 @@ public static ReferenceManager instance() {
             return staticManager;
         }
     }
+
+    public static void clearInstance() {
+        threadLocalManager.set(null);
+        staticManager = null;
+    }
```

Comment:
how is this change related?

Reply:
This is to [fix tests in Formplayer](https://github.com/dimagi/formplayer/pull/1125/commits/e9d372a96180272184c9dd05984f52da5a512990).  For test cases using unarchived ccz file, we need to clear references to staticManager for app to correctly install.